iothub/service/src/Http2/HttpMessageHelper2.cs

```diff
@@ -69,11 +69,10 @@ internal static async Task ValidateHttpResponseStatusAsync(HttpStatusCode expect
         /// </summary>
         /// <typeparam name="T">The type of object to deserialize the response payload into.</typeparam>
         /// <param name="response">The HTTP response containing the payload to deserialize.</param>
-        /// <param name="cancellationToken">Cancellation token.</param>
         /// <returns>The deserialized object.</returns>
-        internal static async Task<T> DeserializeResponseAsync<T>(HttpResponseMessage response, CancellationToken cancellationToken)
+        internal static async Task<T> DeserializeResponseAsync<T>(HttpResponseMessage response)
```

Comment:
Most of the files changed in this PR are because this function no longer needs a cancellation token.